ddtrace/ddtrace.stub.php

```diff
@@ -0,0 +1,1051 @@
+
+    /**
+     * Serializes and sends traces to the agent (in the format dd_trace_serialize_closed_spans() returns spans).
+     *
+     * @internal
+     * @param array $traceArray Array in the format returned by dd_trace_serialize_closed_spans()
+     */
+    function dd_trace_buffer_span(array $traceArray): bool {}
+
+    /**
+     * Used to send any already buffered spans to the agent
+     *
+     * @internal
+     * @return int
+     */
+    function dd_trace_coms_trigger_writer_flush(): int {}
+
+    /**
+     * Execute a given internal function
+     *
+     * Internal functions are: init_and_start_writer, ddtrace_coms_next_group_id, ddtrace_coms_buffer_span,
+     * ddtrace_coms_buffer_data, shutdown_writer, set_writer_send_on_flush, test_consumer, test_writers,
+     * test_msgpack_consumer, synchronous_flush, and root_span_add_tag
+     *
+     * @internal
+     * @param string $functionName Internal function name
+     * @param mixed $args,... Arguments of the function
+     * @return mixed false if void function was properly executed, else the return value of it
+     */
+    function dd_trace_internal_fn(string $functionName, mixed ...$args) {}
+
+    /**
+     * Set the distributed trace id
+     *
+     * @param string|null $traceId New trace id
+     * @return bool 'true' if the change was properly applied, else 'false'
+     */
+    function dd_trace_set_trace_id(?string $traceId = null): bool {}
+
+    /**
+     * Tracks closed spans
+     *
+     * @return int Number of closed spans
+     */
+    function dd_trace_closed_spans_count(): int {}
+
+    /**
+     * Check if the tracer's current memory usage is higher than the set limits
+     *
+     * @return bool 'true' if memory is overused, else 'false'
+     */
+    function dd_trace_tracer_is_limited(): bool {}
+
+    /**
+     * Get the compiling time of all files compiled up to now (in µs)
+     *
+     * @return int Compile time
+     */
+    function dd_trace_compile_time_microseconds(): int {}
+
+    /**
+     * Get serialized information about closed spans as arrays. Note that calling this function will result in
+     * automatically closing unfinished spans (destroys the open span stack).
+     *
+     * @return array Closed spans data
+     */
+    function dd_trace_serialize_closed_spans(): array {}
+
+    /**
+     * Get the currently active span id, or the distributed parent trace id if there is no currently active span
+     *
+     * @return string Currently active span/trace unique identifier
+     */
+    function dd_trace_peek_span_id(): string {}
+
+    /**
+     * Force-finish all spans and force-send finished traces to the agent. Note that this function drops (resp. closes)
+     * all currently open spans if DD_AUTOFINISH_SPANS is set to 'false' (resp. 'true'). Dropped spans are not sent to
+     * the agent.
+     */
+    function dd_trace_close_all_spans_and_flush(): void {}
+
+    /**
+     * @alias DDTrace_trace_function
```

Comment:
Could you please change `@alias` tag to `@see`. Stub tests check PHPDoc tags based on `canonical` list of tags https://docs.phpdoc.org/3.0/guide/guides/docblocks.html#more-on-docblocks and these tags don't contain `@alias`